userguide/content/en/docs/adding-content/lookandfeel.md

```diff
@@ -292,6 +274,36 @@ of the following:
 [wordmark]: https://en.wikipedia.org/wiki/Wordmark
 [your logo]: /docs/adding-content/iconsimages/#add-your-logo
 
+### Light/Dark mode menu
+
+To enable the display of a Light-/[Dark-mode] menu in the navbar, set
+`params.ui.showLightDarkModeMenu` to true in your project's configuration file.
```

Comment:
`params.ui.showLightDarkModeMenu`:

In the section [`Code highlighting with Prism`](https://deploy-preview-1914--docsydocs.netlify.app/docs/adding-content/lookandfeel/#code-highlighting-with-prism) above, we put a tabbed pane in place, showing `toml`, `yaml` and `json` variants. For the sake of consistency, I would propose to implement the same here.

Reply:
@LisaFC et al.: we need to discuss this in the next PSC. I find that for simple boolean params, it isn't worth overhead. If we had something setup like what they have for the Hugo docs, it would make it slightly more bearable!  Anyhow, I've made the addition this time.